setup/aggregator/docker-compose.yml

```diff
@@ -1,11 +1,17 @@
 version: "3"
 
+networks:
+  near-sffl-aggregator:
```

Comment:
I'd prefer if this had a different name to avoid confusion. Maybe a `-net` suffix?

Reply:
Confusion with what?

Reply:
The container name

Reply:
Same applies to relayer btw

Reply:
Don't really see in what scenario there shall be a confusion.

```
networks:
  near-sffl-aggregator:
    driver: bridge
```
In here it's obvious that ist network. Working from cli requires --network flag or network subcommand. 
It sounds smth like "wet water"

Reply:
I prefer leaving it the way it is. Can change if it will be confusing indeed in some scenario

Reply:
Well, it gets weird imo since there's already a container with the exact same name, and the network is not used only on it

Reply:
But feel free to leave as it is, it's not blocking